.github/workflows/release.yml

```diff
@@ -5,8 +5,10 @@ on:
     types: [published]
 
 jobs:
-  deploy:
+  build:
+    name: Build
     runs-on: ubuntu-latest
+    environment: Live Testing
```

Comment:
Why not split this into build and test as well?

Reply:
Release will depend on live tests and live test should only run if unit tests have passed. As there's no parallel work, I didn't see the need to run this in a separate job. 

The same could be said for the Deploy job.   I don't see any reason for it to be its own job.

Reply:
... except for retries.   I would be handy to be able to retry a deployment after a long live test run